modules/tinymce/src/plugins/template/test/ts/browser/TemplateSanityTest.ts

```diff
@@ -78,4 +75,27 @@ describe('browser.tinymce.plugins.template.TemplateSanityTest', () => {
     editor.execCommand('mceInsertTemplate', false, '<p>{$first+name}</p>');
     TinyAssertions.assertContent(editor, '<p>Tester</p>');
   });
+
+  it('TINY-9244: Parsed html should be shown when previewing templates', async () => {
+    const editor = hook.editor();
+    const unparsedHtml = '<img src="error" onerror="console.log(`Unparsed html`);">';
+    addSettings({
+      templates: [{ title: 'a', description: 'b', content: unparsedHtml }],
+    });
+
+    const unParsedPreviewHtmlSelector = 'p > img[src="error"][onerror="console.log(`Unparsed html`);"]';
+    const parsedPreviewHtmlSelector = 'p > img[src="error"][data-mce-src="error"]';
+    const assertPreviewContent = (dialogEl: SugarElement<Node>, existsSelector: string, notExistsSelector: string): void => {
+      UiFinder.findIn<HTMLIFrameElement>(dialogEl, 'iframe').fold(
+        () => assert.fail('Preview iframe not found'),
+        (iframe) => {
+          const iframeDoc = iframe.dom.contentDocument || iframe.dom.contentWindow?.document;
```

Comment:
Why are there two cases here will not contentDocument work on all browsers? If not add a comment why and on what.

Reply:
The code was there before my change. I assume this is because contentDocument isn't supported on pre- IE8 versions of IE. Added comment to document that.

Reply:
Oh is it that old then get rid of it we don't support IE at all anymore.